components/ActionButton.tsx

```diff
@@ -32,9 +31,9 @@ const ActionButton: FC<ActionButtonProps> = ({
   }
 
   return href ? (
-    <Link href={href}>
-      <a className={classStyle}>{text}</a>
-    </Link>
+    <a href={href} className={classStyle}>
+      {text}
+    </a>
```

Comment:
What I mean is all this can be removed, along with the href prop (if that's desired the LinkButton should be used)